.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 dist/
 yarn-error.log
+package-lock.json
```

Comment:
Lockfiles should not be added to gitignore as they allow to pin specific versions of dependencies, preventing to download different versions when running 2 times "npm install" at different times but from the same source code